.github/workflows/publish-coverage-report.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish coverage report to Github Pages
+
+on:
+  workflow_run:
+    workflows: ["websocket: build/host-tests", "esp-modem: build/host-tests"]
+    types:
+      - completed
+
+jobs:
+  publish_github_pages:
+    needs: [host_test_websocket, host_test_gcov_esp_modem]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Download Artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: docs_gcovr_modem
+          path: $GITHUB_WORKSPACE/artifact1
```

Comment:
could we give it a better name? instead of `artifact1` something like `modem_coverage_report`, or similar?